javalib/src/main/scala/java/math/BigInteger.scala

```diff
@@ -462,6 +462,10 @@ class BigInteger extends Number with Comparable[BigInteger] {
   override def doubleValue(): Double =
     java.lang.Double.parseDouble(this.toString)
 
+  @inline override def __scala_==(other: _Object): scala.Boolean = {
+    this.equals(other)
+  }
+
```

Comment:
Same as the comment in `BigDecimal` above.

Reply:
removed